components/admin/requests/processing/review-information.ts

```diff
@@ -0,0 +1,26 @@
+import { gql } from '@apollo/client'; // GraphQL queries
+import { Application, Applicant, QueryApplicationArgs } from '@lib/graphql/types';
+
+// Queries an Application by ID along with the associated permit, replacement, applicationProcessing, and applicant
+export const GET_REVIEW_INFORMATION_QUERY = gql`
+  query GetApplication($id: Int!) {
+    application(id: $id) {
+      id
+      type
+      applicant {
+        guardian
+      }
+    }
```

Comment:
We don't want to query the guardian via the applicant, as that is already on record. We want to pull the guardian information filled out in the application itself. Instead, we want to query the following:
```suggestion
    application(id: $id) {
      __typename
      id
      type
      ... on NewApplication {
        guardianFirstName
        guardianMiddleName
        guardianLastName
        etc...
      }
    }
```
which uses [GraphQL fragments](https://graphql.org/learn/queries/#fragments) to only query the guardian fields when we are dealing with a new application